runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -398,6 +398,11 @@ public void onGameTick(final GameTick event)
 					}
 				}
 			}
+
+			if (!npcHintArrowMarked && !addedHintArrow)
+			{
+				client.clearHintArrow();
```

Comment:
can't you just move this up above L380 and avoid the whole "addedHintArrow" logic?

Reply:
I guess there's no harm in clearing the hint arrow each tick before resetting it. :+1: